### DISCUSSION TITLE
4.6.30

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#601] [Announcements]
#### ADDED
- 新增导入导出设置功能
- 支持F5或Ctrl+R快捷键刷新页面
- 支持设置保持音量与亮度
#### CHANGED
- 首页动态页支持查看评论区图片
- 快捷键切换播放倍速时提示当前倍速
- 播放器内消息提示位置及动画调整
- 播放列表切换视频时检测低音量并提示
- 播放完毕后上报历史记录位置0使其他平台打开该视频为重新开始播放
- 导出视频时自动设置文件名为视频标题+分p标题
- 直播详情页保存显示底部礼物栏设置
#### FIXED
- 更换首页动态页接口并重新适配数据，解决打开错误问题
- 修复首页推荐页视频链接解析错误

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.30'>4.6.30</a>.</em>

**Comment by lyppiao** [comment]
可以加个保留历史纪录位置的开关吗

**Reply by ywmoyue** [reply]
什么意思呢，是想要在uwp中播放视频后不在历史记录中可查吗

**Reply by ywmoyue** [reply]
还是说针对这一条变更希望加个设置项 
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/67c50d2a-7ad2-47f0-b253-4f3e781de830)

**Comment by lyppiao** [comment]
多个设置，可以像旧版一样保留播放进度
